learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -127,17 +127,19 @@ service / on httpListener {
 }
 ```
 
-Let's take a moment to digest the new constructs in this code:
+Let's take a moment to digest the new constructs below in this code:
```

Comment:
There is no code `below`. The code is above the description.

Reply:
+1